$MirOS: ports/lang/llvm/patches/patch-lib_Support_CommandLine_cpp,v 1.1 2008/11/10 01:01:32 tg Exp $

	Support branding, especially via CPPFLAGS

--- lib/Support/CommandLine.cpp.orig	Thu Jan  1 01:00:00 1970
+++ lib/Support/CommandLine.cpp	Fri Dec  4 20:44:07 2009
@@ -1037,6 +1037,10 @@ static int OptNameCompare(const void *LH
   return strcmp(((pair_ty*)LHS)->first, ((pair_ty*)RHS)->first);
 }
 
+#ifdef LLVM_VERSION_INFO_SVN
+#define LLVM_VERSION_INFO " SVN " LLVM_VERSION_INFO_SVN
+#endif
+
 namespace {
 
 class HelpPrinter {
